Makefile isn't able to update the binary

output when running after a git pull

    # make install
    echo installing
    installing
    mkdir -p /usr/local/lib/brightnessctl
    cp -R ./*  /usr/local/lib/brightnessctl/
    ln -s /usr/local/lib/brightnessctl/writebrightness.sh /usr/local/bin/brightness
    ln: failed to create symbolic link '/usr/local/bin/brightness': File exists
    makefile:4: recipe for target 'install' failed
    make: *** [install] Error 1

--- a/makefile
+++ b/makefile
@@ -4,9 +4,9 @@ install:
 	echo installing
 	mkdir -p $(INSTALL_FOLDER)
 	cp -R ./*  $(INSTALL_FOLDER)/
-	ln -s $(INSTALL_FOLDER)/writebrightness.sh $(BIN_FOLDER)/brightness
-	ln -s $(INSTALL_FOLDER)/increase.sh $(BIN_FOLDER)/brightness+
-	ln -s $(INSTALL_FOLDER)/decrease.sh $(BIN_FOLDER)/brightness-
+	ln -fs $(INSTALL_FOLDER)/writebrightness.sh $(BIN_FOLDER)/brightness
+	ln -fs $(INSTALL_FOLDER)/increase.sh $(BIN_FOLDER)/brightness+
+	ln -fs $(INSTALL_FOLDER)/decrease.sh $(BIN_FOLDER)/brightness-
 	echo "don't forget to modify your sudoers file and i3 config"
 
 uninstall:

